openapi-cli/src/main/java/io/ballerina/openapi/cmd/OpenApiCmd.java

```diff
@@ -104,9 +104,12 @@ public class OpenApiCmd implements BLauncherCmd {
     private boolean includeTestFiles;
 
     @CommandLine.Option(names = {"--client-methods"}, hidden = true, description = "Generate the client methods" +
-            " with provided type . Only \"remote\"(default) and \"resource\" options are supported.")
+            " with provided type . Only \"resource\"(default) and \"remote\" options are supported.")
     private String generateClientMethods;
 
+    @CommandLine.Option(names = {"--with-service-type"}, hidden = true, description = "Generate service type")
```

Comment:
Shall we handle the scenario, if the user has entered the command option `--with-service-type` with client generation mode ex: `bal openapi -i <yaml file> --mode client --with-service-type`?

Reply:
How do u suggest to handle it ? In the current way it will be ignored for client generation.

Reply:
@aneeshafedo @lnash94 IMO no need to worry about this scenario for now, as the command option is temporary and expected to be replaced in the next release anyway. Therefore as long as we gracefully ignore the service type generation in client mode, we should be fine

Reply:
Shall we provide the warning or error by mentioning service type flag cannot be used in client generation mode :) ?

Reply:
Currently we have decided to hide the command option anyway from the users since it will be removed from the next release anyway.

Reply:
+1